### PR TITLE
First attempt to load Zip codes for our 14 counties.

### PR DESCRIPTION
**Pull request by makfan64**
Hi Vince, here is the first draft of the zip code table load.

### CODE REVIEW

**Review by VincentLa14 — COMMENTED** [review]
This is mostly good!

File: pipeline/pipeline_tasks/parse/00_04_01_usps_zips.sql [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+create table stg_analytics.sba_sfdo_zips
+(
+zip_code text,
+zip_city text,
+zip_state text,
+zip_county text
+);
+
+insert into stg_analytics.sba_sfdo_zips
+values (
```
Comment (VincentLa14) [inline pipeline/pipeline_tasks/parse/00_04_01_usps_zips.sql]: @makfan64 Be careful here, this doesn't return what you think this returns. When using the enclosing parentheses in values you're denoting different rows. Inside those parentheses you're denoting different columns. For example,

```
values (
('a', 'b'),
('c', 'd')
)
```
will actually return one row with two columns:
```
column1   column2
('a', 'b')   ('c', 'd')
```
However, 

```
values
('a', 'b'),
('c', 'd')
```
Returns two columns with two rows
```
column1           column2
'a'           'b'
'c'           'd'
```
which is what you actually intend here.

Long story short, I would remove the initial enclosing parentheses here.

Comment (makfan64) [reply pipeline/pipeline_tasks/parse/00_04_01_usps_zips.sql]: Thank you. I tried looking this up but still got it wrong. I will fix.

**Comment by VincentLa14** [comment]
Awesome looks great to me.

**Comment by VincentLa14** [comment]
LGTM (Looks good to me!) Feel free to merge. Really sorry about this circleci check failure thing. Don't worry about it. I'm just trying to set up an automated testing framework for us, but haven't nailed it down yet.

**Review by VincentLa14 — APPROVED** [review]

**Pull request merged**